src/components/CommandLineComponent.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+    <div class="col command-line">
+      <div>
+        <q-editor
+          v-model="editor"
+          min-height="1rem"
+          max-height="5rem"
+          flat
+          overflow-hidden
+          toolbar-bg="grey"
+          :toolbar="[
+          ['bold', 'italic', 'strike'],
+          ['link'],
+          ['unordered', 'ordered'],
+          ['quote'],
+        ]"
+        />
+      </div>
+      <div class="down-menu-bar">
+        <q-btn round dense flat icon="send" size="sm" class="q-ma-xs"/>
+      </div>
+    </div>
+</template>
+
+<script>
+</script>
+
+<style scoped>
+  /* Input block */
+  .command-line {
+    background: #FFFFFF;
+    border: 1px solid #00000015;
+    border-radius: 10px;
+    margin-bottom: 0.5rem;
+  }
+
+  /* Input block modifier */
+  @media (max-width: 1024px) {
```

Comment:
tieto media queries idealne spravit cez quasar classy, nech to nemame rozhadzane halabala, ak to pojde

Reply:
Je to stále riešené cez media queries, pretože quasar neponúka classu pre max-width, aspoň teda som nenašiel keď tak opravím potom. Zatial nechám takto.